tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs

```diff
@@ -846,148 +1126,67 @@ public static void TestSpanSliceStringChar(int length)
             }
             string s = sb.ToString();
 
-            foreach (var iteration in Benchmark.Iterations)
-                using (iteration.StartMeasurement())
-                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
-                    {
-                        ReadOnlySpan<char> temp = s.Slice();
-                    }
+            Invoke((int innerIterationCount) => TestSpanSliceStringChar(s, innerIterationCount, false),
+                "TestSpanSliceStringChar({0})", length);
         }
-        #endregion
 
-        #endregion
-        
-        // EXE-based testing
-        #region EXE-base testing
-        /*static void FillAllSpanBase()
-        {
-            byte[] a = new byte[Size];
-            Span<byte> s = new Span<byte>(a);
-            for (int i = 0; i < FillAllIterations; i++)
-            {
-                TestFillAllSpan(s);
-            }
-        }
-
-        static void FillAllArrayBase()
-        {
-            byte[] a = new byte[Size];
-            for (int i = 0; i < FillAllIterations; i++)
-            {
-                TestFillAllArray(a);
-            }
-        }
-
-        static void FillAllReverseSpanBase()
-        {
-            byte[] a = new byte[Size];
-            Span<byte> s = new Span<byte>(a);
-            for (int i = 0; i < FillAllIterations; i++)
-            {
-                TestFillAllReverseSpan(s);
-            }
-        }
-
-        static void FillAllReverseArrayBase()
-        {
-            byte[] a = new byte[Size];
-            for (int i = 0; i < FillAllIterations; i++)
-            {
-                TestFillAllReverseArray(a);
-            }
-        }
-
-        static void QuickSortSpanBase()
-        {
-            int[] data = new int[Size];
-            int[] unsortedData = GetUnsortedData();
-            Span<int> span = new Span<int>(data);
-
-            for (int i = 0; i < QuickSortIterations; i++)
-            {
-                Array.Copy(unsortedData, data, Size);
-                TestQuickSortSpan(span);
-            }
-        }
-
-        static void BubbleSortSpanBase()
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        static void TestSpanSliceStringChar(string s, int iterationCount, bool untrue)
```

Comment:
In the other tests, the [Benchmark] method is suffixed with a type and the kernel method is not.  Since the kernel method isn't generic in this case, there wasn't a type to suffix the [Benchmark] method with, so I added a "Wrapper" suffix to it.  It's a bit easier to inspect jitting of the kernel methods if they aren't overloads, is why I bothered.  That said, if you want them to have the same name, I'll change it, though wouldn't it be better to remove "Wrapper" from the other method?